internal/lvgen/generate.go

```diff
@@ -159,13 +159,45 @@ type Case struct {
 
 // Proc holds information about a libvirt procedure the parser has found.
 type Proc struct {
-	Num        int64  // The libvirt procedure number.
-	Name       string // The name of the go func.
-	LVName     string // The name of the libvirt proc this wraps.
-	Args       []Decl // The contents of the args struct for this procedure.
-	Ret        []Decl // The contents of the ret struct for this procedure.
-	ArgsStruct string // The name of the args struct for this procedure.
-	RetStruct  string // The name of the ret struct for this procedure.
+	Num            int64  // The libvirt procedure number.
+	Name           string // The name of the go func.
+	LVName         string // The name of the libvirt proc this wraps.
+	Args           []Decl // The contents of the args struct for this procedure.
+	Ret            []Decl // The contents of the ret struct for this procedure.
+	ArgsStruct     string // The name of the args struct for this procedure.
+	RetStruct      string // The name of the ret struct for this procedure.
+	ReadStreamIdx  int    // The index of read stream in function argument list
+	WriteStreamIdx int    // The index of read stream in function argument list
+}
+
+// ProcMetaGenerate is a type for flags from @generate annotation
+type ProcMetaGenerate int
+
+// @generate annotation can be none, client, server or both
+const (
+	ProcMetaGenerateNone ProcMetaGenerate = iota
+	ProcMetaGenerateClient
+	ProcMetaGenerateServer
+	ProcMetaGenerateBoth
+)
+
+// ProcMetaPriority is a type for values from @priority annotation
+type ProcMetaPriority int
+
+// @priority annotation can be low or high
```

Comment:
what does high mean? what dose low mean? this comment doesn't help future readers.

Reply:
tbh, these types won't help future readers as well. I guess I better get rid of them.